ticdc/ticdc-changefeed-config.md

```diff
@@ -204,6 +204,8 @@ enable-partition-separator = true
 # output-handle-key = false
 # Whether to output the value before the row data changes. The default value is false. 
 # When it is enabled, the UPDATE event will output two rows of data: the first row is a DELETE event that outputs the data before the change; the second row is an INSERT event that outputs the changed data.
+# When it is enabled (setting it to true), the "is-update" column will be added before the column with data changes. This added column is used to identify whether the data change of the current row comes from the UPDATE event or the original INSERT/DELETE event.
+# If the data change of the current row comes from the UPDATE event, the value of the "is-update" column is true, otherwise it is false.
```

Comment:
```suggestion
# If the data change of the current row comes from the UPDATE event, the value of the "is-update" column is true. Otherwise it is false.
```